src/views/catalog/customize/components/CustomizeForm.tsx

```diff
@@ -41,7 +59,13 @@ export const CustomizeForm: React.FC<CustomizeFormProps> = ({ template }) => {
       if (requiredFieldNoValue) {
         setValidationError(true);
       } else {
-        await createVirtualMachine(template, ns, parameterForName, formData);
+        await createVirtualMachine(
```

Comment:
you are not creating a virtual machine. you are creating a processed template.
please change the name of the function